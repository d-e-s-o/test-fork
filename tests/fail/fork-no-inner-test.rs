// Copyright (C) 2025 Daniel Mueller <deso@posteo.net>
// SPDX-License-Identifier: (Apache-2.0 OR MIT)

#[test_fork::fork]
fn missing_test() {}

fn main() {}
