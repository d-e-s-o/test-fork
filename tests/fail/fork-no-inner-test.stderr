error: test_fork::fork requires an inner #[test] attribute
 --> tests/fail/fork-no-inner-test.rs:4:1
  |
4 | #[test_fork::fork]
  | ^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `test_fork::fork` (in Nightly builds, run with -Z macro-backtrace for more info)
